cpp/src/arrow/adapters/arrow-rados-cls/cls_arrow.cc

```diff
@@ -78,23 +90,36 @@ class RandomAccessObject : public arrow::io::RandomAccessFile {
     return std::make_shared<arrow::Buffer>("");
   }
 
+  /// \brief Read the byte stream.
+  /// \param[in] nbytes Number of bytes.
+  /// \return Buffer of the objects in the stream.
```

Comment:
Same, no docstring needed for RadosObjectClass methods

Reply:
Done.